tools/apiview/parsers/cpp-api-parser/ParseAzureSdkCpp/CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ add_executable(ParseAzureSdkCpp ParseAzureSdkCpp.cpp )
 add_dependencies(ParseAzureSdkCpp ApiViewProcessor)
 target_include_directories(ParseAzureSdkCpp PRIVATE ${ApiViewProcessor_SOURCE_DIR})
 
+#if(MSVC)
+target_link_options(ParseAzureSdkCpp PRIVATE /STACK:2000000)
```

Comment:
Can you add the comment from the PR info into source which explains why we need to adjust the stack

Reply:
Done.